lib/logmsg/logmsg.h

```diff
@@ -450,6 +450,12 @@ log_msg_set_value_by_name(LogMessage *self, const gchar *name, const gchar *valu
   log_msg_set_value_by_name_with_type(self, name, value, length, LM_VT_STRING);
 }
 
+static inline void
+log_msg_set_value_to_string(LogMessage *self, NVHandle handle, const gchar *literal_string)
+{
+  log_msg_set_value(self, handle, literal_string, strlen(literal_string));
```

Comment:
We call this on the hot path, for every message, when setting the `$TRANSPORT` macro.
I think it could be better if we'd call `log_msg_set_value()` with `sizeof()` or introduce `log_msg_set_value_to_literal_string` as a macro to avoid this per-message `strlen()` call.

Reply:
I didn't check but this should be inlined with the same effect.

Hmm https://stackoverflow.com/questions/1733281/where-is-the-implementation-of-strlen-in-gcc/50199212#50199212

It may not, unless we compile using -O3